### PR TITLE
Issue/har 97/create event and point form

### PR DESCRIPTION
**Pull request by Bedpaw**

### CODE REVIEW

**Comment by dbetka** [comment]
Lack of visible validation below form fields.

Screenshot 2023-05-02 at 21 02 34
Screenshot 2023-05-02 at 21 03 10

**Comment by dbetka** [comment]
After adding point form is not resetting its state.

Screenshot 2023-05-02 at 21 48 15

**Comment by dbetka** [comment]
Allow select to take string or number via model value prop

Screenshot 2023-05-02 at 21 01 01

**Review by dbetka — CHANGES_REQUESTED** [review]

**Comment by Bedpaw** [comment]
> Allow select to take string or number via model value prop
> 
> Screenshot 2023-05-02 at 21 01 01
Fixed

**Comment by Bedpaw** [comment]
> Lack of visible validation below form fields (text and date fields and point position).
> 
> Screenshot 2023-05-02 at 21 02 34Screenshot 2023-05-02 at 21 03 10

Done for all expect event position, it require some discussion

File: client/src/features/map/map-position-picker.vue
[the remark this answers is not in the document]
Comment (Bedpaw) [reply client/src/features/map/map-position-picker.vue:1]: done, but not sure if eslint applied as expected

**Comment by Bedpaw** [comment]
> After adding point form is not resetting its state.
> 
> Screenshot 2023-05-02 at 21 48 15

Fixed

**Review by dbetka — APPROVED** [review]

**Pull request merged**